### PR TITLE
Add missing content to the Docker docs (#1804)

### PR DESCRIPTION
**Pull request by renetapopova**
https://trello.com/c/MgmtbjRJ/1470-support-docker-secrets

* Adds documentation for general docker compose usage
* Adds documentation for using secrets with neo4j

---------

Cherry-picked from #1804 

* Add documentation for persistent volumes as in 5.x
* Add documentation for plugins as in 5.x

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1916.surge.sh

New pages:
- [Deploy a Neo4j standalone server using Docker Compose](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/docker-compose-standalone)

Updated pages:
- [Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1916.surge.sh

New pages:
- [Deploy a Neo4j standalone server using Docker Compose](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/docker-compose-standalone)

Updated pages:
- [Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1916.surge.sh

New pages:
- [Deploy a Neo4j standalone server using Docker Compose](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/docker-compose-standalone)
- [Persisting data with Docker volumes](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/mounting-volumes)
- [Plugins](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/plugins)

Updated pages:
- [Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker)

File: modules/ROOT/pages/docker/docker-compose-standalone.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+. Create a project folder where you will store your _docker-compose.yml_ file and run your Neo4j server.
+. Prepare your _docker-compose.yml_ file using the following example.
+For more information, see the Docker Compose official documentation on https://docs.docker.com/compose/compose-file/#service-configuration-reference[Docker Compose specification].
++
+.Example of a _docker-compose.yml_ file
+[source,yaml,subs="attributes+,+macros"]
+----
+services:
+  neo4j:
+    image: neo4j:latest
```
Comment (stefgia) [inline modules/ROOT/pages/docker/docker-compose-standalone.adoc:24]: This version should probably be `{neo4j-version-exact}` like in the other sections? Or the latest 4.4

File: modules/ROOT/pages/docker/docker-compose-standalone.adoc [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+----
+. Prepare your _docker-compose.yml_ file using the following example.
+For more information, see the Docker Compose official documentation on https://docs.docker.com/compose/compose-file/#service-configuration-reference[Docker Compose specification].
++
+.Example of a _docker-compose.yml_ file
+[source,yaml,subs="attributes+,+macros"]
+----
+services:
+  neo4j:
+    image: neo4j:latest
```
Comment (stefgia) [inline modules/ROOT/pages/docker/docker-compose-standalone.adoc:71]: Same for this version

Comment (renetapopova) [reply modules/ROOT/pages/docker/docker-compose-standalone.adoc:71]: Good catch!

File: modules/ROOT/pages/docker/docker-compose-standalone.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/docker/docker-compose-standalone.adoc]: Yes, it should be `{neo4j-version-exact}`.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1916.surge.sh

New pages:
- [Deploy a Neo4j standalone server using Docker Compose](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/docker-compose-standalone)
- [Persist data with Docker volumes](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/mounting-volumes)
- [Plugins](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/plugins)

Updated pages:
- [Deploy a Neo4j cluster on Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/clustering)
- [Modify the default configuration](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/configuration)
- [Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker)
- [Get started with Neo4j in Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/introduction)
- [Docker specific operations](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/operations)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1916.surge.sh

New pages:
- [Deploy a Neo4j standalone server using Docker Compose](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/docker-compose-standalone)
- [Persist data with Docker volumes](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/mounting-volumes)
- [Plugins](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/plugins)

Updated pages:
- [Deploy a Neo4j cluster on Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/clustering)
- [Modify the default configuration](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/configuration)
- [Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker)
- [Get started with Neo4j in Docker](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/introduction)
- [Docker specific operations](https://neo4j-docs-operations-1916.surge.sh/operations-manual/4.4/docker/operations)

**Review by stefgia — APPROVED** [review]

**Pull request merged**